docs/DeveloperGuide.md

```diff
@@ -172,6 +172,32 @@ Step 5. During the execution of the `AddProjectCommand`, selected `Employee` obj
 
 Step 6. A `CommandResult` is produced based on whether the execution was a success or not and returned to the `LogicManager`. 
 
+### Prioritising Projects Feature
```

Comment:
`Prioritise` would be better than `Prioritising`

Reply:
Sure!